build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 // please update RELEASE_NOTES.md when you update this version
-def buildVersion = "1.5.6"
+def buildVersion = "1.6.0"
```

Comment:
Need to update release notes

Reply:
Release notes are updated 
<img width="1399" alt="image" src="https://github.com/govuk-one-login/ipv-cri-lib/assets/40401118/9ff3f645-2b14-4b68-ade0-20efd45bc571">